.travis.yml

```diff
@@ -61,6 +61,7 @@ git:
   quiet: true
 
 install:
+  - rm -f ~/.m2/settings.xml
```

Comment:
uh... why? Please leave some explanation in code comment

Reply:
what is set there by default?

Reply:
Sorry, was a hack to allow the build to run, not intended to be permanent. Travis uses the Google mirror of Central, but the artifact hadn't synced yet after over 24 hours. I filed an issue with Sonatype to investigate.

This is what their default config file looks like:
```xml
<settings>
  <profiles>
    <profile>
      <id>standard-with-extra-repos</id>
      <activation>
        <activeByDefault>true</activeByDefault>
      </activation>
      <repositories>
        <repository>
          <id>central</id>
          <name>Central Repository</name>
          <url>https://repo.maven.apache.org/maven2</url>
          <releases>
            <enabled>true</enabled>
          </releases>
          <snapshots>
            <enabled>false</enabled>
          </snapshots>
        </repository>
        <repository>
          <id>sonatype</id>
          <name>OSS Sonatype repo (releases)</name>
          <releases>
            <enabled>true</enabled>
            <updatePolicy>always</updatePolicy>
            <checksumPolicy>warn</checksumPolicy>
          </releases>
          <snapshots>
            <enabled>false</enabled>
            <updatePolicy>never</updatePolicy>
            <checksumPolicy>fail</checksumPolicy>
          </snapshots>
          <url>https://oss.sonatype.org/content/repositories/releases/</url>
        </repository>
        <repository>
          <id>sonatype-snapshots</id>
          <name>OSS Sonatype repo (snapshots)</name>
          <releases>
            <enabled>false</enabled>
            <updatePolicy>always</updatePolicy>
            <checksumPolicy>warn</checksumPolicy>
          </releases>
          <snapshots>
            <enabled>true</enabled>
            <updatePolicy>never</updatePolicy>
            <checksumPolicy>fail</checksumPolicy>
          </snapshots>
          <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
        </repository>
        <repository>
          <id>sonatype-apache</id>
          <name>Apache repo (releases)</name>
          <releases>
            <enabled>true</enabled>
            <updatePolicy>always</updatePolicy>
            <checksumPolicy>warn</checksumPolicy>
          </releases>
          <snapshots>
            <enabled>false</enabled>
            <updatePolicy>never</updatePolicy>
            <checksumPolicy>fail</checksumPolicy>
          </snapshots>
          <url>https://repository.apache.org/content/repositories/releases/</url>
        </repository>
        <repository>
          <id>apache-snapshots</id>
          <name>ASF repo (snapshots)</name>
          <releases>
            <enabled>false</enabled>
            <updatePolicy>never</updatePolicy>
            <checksumPolicy>warn</checksumPolicy>
          </releases>
          <snapshots>
            <enabled>true</enabled>
            <updatePolicy>always</updatePolicy>
            <checksumPolicy>fail</checksumPolicy>
          </snapshots>
          <url>https://repository.apache.org/snapshots/</url>
        </repository>
      </repositories>
    </profile>
  </profiles>
<mirrors>
    <mirror>
      <id>google-maven-central</id>
      <name>GCS Maven Central mirror</name>
      <url>https://maven-central.storage-download.googleapis.com/repos/central/data/</url>
      <mirrorOf>central</mirrorOf>
    </mirror>
  </mirrors></settings>
```